src/store/modules/user.js

```diff
@@ -0,0 +1,32 @@
+import FireBaseService from "../../services/authfirebase";
+
+const user = {
+  state: {
+    user: {},
+    errors: {},
+  },
+  getters: {
+    getUserData: state => state.user,
+    getUserErrors: state => state.errors,
+  },
+  mutations: {
+    setUserData: (state, payload) => {
+      state.user = payload.user;
+    },
+    setUserErrors: (state, payload) => {
```

Comment:
@daktari01 i advised you make you of a generic error component, which will have its own store module, i will be doing this in the next session.

Reply:
Thanks for the advice. I will make the changes after the lesson. However, I request to go ahead and merge so that I can work on the `sign in` feature.